configuration-modes/appsensor-configuration-stax/src/test/resources/appsensor-server-config.xml

```diff
@@ -29,152 +29,191 @@
 			<client-application-name>server5</client-application-name>
 		</correlated-client-set>
 	</correlation-config>
+	
+	
 
-	<detection-points>
```

Comment:
@shreyasdn not sure what the change is here, looks like this is all the same as what's below with only the change of the addition of the <clients> element. Just want to make sure there's nothing I'm missing here.